cpp/log/test.cpp

```diff
@@ -59,6 +64,104 @@ void test() {
 
 	t0.join();
 	t1.join();
+
+	return 0;
+}
+
+
+/**
+ * Creates a FileSink to also write log Messages
+ * into /tmp/openage-log-test.
+ * Only Messages with a level equals or higher
+ * than level::err will be printed in that file.
+ */
+int file_log_sink () {
+	char filelog_name[L_tmpnam];
+	int result = 0;
+
+	tmpnam(filelog_name);
```

Comment:
If you want/must stick to generating a filename only, you should use at least use [tempnam](http://linux.die.net/man/3/tempnam) instead.